dvc/api.py

```diff
@@ -92,6 +94,178 @@ def read(path, repo=None, rev=None, remote=None, mode="r", encoding=None):
         return fd.read()
 
 
+def get_params(
+    *targets: str,
+    repo: Optional[str] = None,
+    revs: Optional[Union[str, Iterable[str]]] = None,
+    deps: bool = False,
+    stages: Optional[Union[str, Iterable[str]]] = None,
```

Comment:
Not sure what's expected in this case:

```console
>>> dvc.api.get_params("src/train.py", stages="train")
{'train': {'min_split': 128, 'n_est': 50, 'seed': 20170428}}
```

I expected it to return an empty dict.

Reply:
> I expected it to return an empty dict.

The `targets`/`stages` interaction didn't exist before the P.R. so I wasn't sure how to implement / what's the most intuitive behavior. 

I guess what you expect makes more sense than the current behavior.

Reply:
Updated to match the expected behavior